alacritty/src/event.rs

```diff
@@ -1546,6 +1506,29 @@ impl Processor {
 
                     info!("Initialisation complete");
                 },
+                // NOTE: try to request a redraw early on when we get a wakeup from the terminal
+                // to minimize the end input latency.
```

Comment:
```suggestion
                // NOTE: Try to request a redraw early on when we get a wakeup from the terminal
                // to minimize the end input latency.
```

I'm not sure what this means? Why are we redrawing on every wakeup to begin with, that just seems inefficient? Terminal wakeups happen 24/7.